src/components/Game/Result/Result.js

```diff
@@ -2,26 +2,35 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
-const ResultView = ({ gameWon, correctWords }) => (
-    <div>
-        {
-            (gameWon)
-                ? 'gratulacje, wygrałeś'
-                : (
-                    `nie udało się, może następnym razem.
-                    poprawne słowa to:
-                    ${correctWords.map((word) => word.toUpperCase()).join(', ')}`
-                )
-        }
-    </div>
-);
+const ResultView = ({ correctWords, submittedAnswer }) => {
```

Comment:
are you going to extract this to a separate route? (doesn't have to be now)
I recommend transforming it to a route at some point in the time

Reply:
yes :)